app/containers/Menu.js

```diff
@@ -0,0 +1,254 @@
+var React = require('react');
+var styles = require('../styles');
+var ReactRouter = require('react-router');
+var Link = ReactRouter.Link;
+var Editor = require('./Editor');
+
+var Menu = React.createClass({
+
+  getInitialState: function () {
+
+    return {
+      lastSort: 'Alphabetical',
+      sort: "Popularity",
```

Comment:
lastSort and sort are ambiguous names... without knowledge of the keys, wouldn't know what they do